tests/test_gc_coro.js

```
@@ -0,0 +1,59 @@
+// Simulate TUI-like behavior: async handler with lots of string allocations
+
+function stripAnsi(str) {
+  return str.replace(/\x1b\[[0-9;]*m/g, '');
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)